frontend/src/features/properties/containers/ParcelDetailContainer.tsx

```diff
@@ -39,44 +40,48 @@ const ParcelDetailContainer: React.FunctionComponent<IParcelDetailContainerProps
   const [currentTab, setCurrentTab] = useState(props.defaultTab ?? ParcelDetailTabs.parcel);
   const [pidSelection, setPidSelection] = useState<IPidSelection>({ showPopup: false, geoPID: '' });
   const [showDeleteDialog, setShowDeleteDialog] = useState(false);
-  const [pin, setPin] = useState<IProperty | undefined>(undefined);
+  const [movingPinNameSpace, setMovingPinNameSpace] = useState<string | undefined>(
+    props.movingPinNameSpace,
+  );
 
   const keycloak = useKeycloakWrapper();
   const dispatch = useDispatch();
   const formikRef = React.useRef<FormikValues>();
 
+  React.useEffect(() => {
+    if (movingPinNameSpace !== undefined) {
+      document.body.className =
+        currentTab === ParcelDetailTabs.parcel ? 'parcel-cursor' : 'building-cursor';
+    }
+    return () => {
+      //make sure to reset the cursor when this component is disposed.
+      document.body.className = '';
+    };
+  }, [currentTab, movingPinNameSpace]);
+
   //Add a pin to the map where the user has clicked.
   useDeepCompareEffect(() => {
     //If we click on the map, create a new pin at the click location.
     if (
+      movingPinNameSpace !== undefined &&
       !!formikRef?.current &&
       !!props.mapClickMouseEvent &&
       ((props.mapClickMouseEvent.originalEvent.timeStamp >=
         (document?.timeline?.currentTime ?? 0) - 500) as boolean)
     ) {
-      setPin({
-        id: Number(props.parcelDetail?.id || 0),
-        latitude: props.mapClickMouseEvent.latlng.lat,
-        longitude: props.mapClickMouseEvent.latlng.lng,
-        propertyTypeId: 0,
-      });
-      formikRef.current.setFieldValue('latitude', props.mapClickMouseEvent.latlng.lat);
-      formikRef.current.setFieldValue('longitude', props.mapClickMouseEvent.latlng.lng);
+      const nameSpace = (movingPinNameSpace?.length ?? 0) > 0 ? `${movingPinNameSpace}.` : '';
```

Comment:
only set the pin if the click event is valid and we have already clicked the drop draft marker button.